customer-models/src/main/java/org/powertac/customer/evcharger/DemandSampler.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2022 by John Collins.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.powertac.customer.evcharger;
+
+import org.apache.commons.configuration2.XMLConfiguration;
+import org.apache.commons.configuration2.ex.ConfigurationException;
+import org.apache.commons.math3.distribution.MixtureMultivariateNormalDistribution;
+import org.apache.commons.math3.distribution.MultivariateNormalDistribution;
+import org.apache.commons.math3.distribution.NormalDistribution;
+import org.apache.commons.math3.linear.Array2DRowRealMatrix;
+import org.apache.commons.math3.util.Pair;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.powertac.common.config.Configurator;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Sampler class which holds the logic of sampling new EV charger demand from
+ * the statistical model.
+ *
+ * @author Philipp Page <github@philipp-page.de>
+ */
+class DemandSampler
+{
+  private static final Logger log =
+    LogManager.getLogger(DemandSampler.class.getSimpleName());
+  private MixtureMultivariateNormalDistribution pluginProbability;
+  private XMLConfiguration config;
+
+  void initialize ()
+  {
+    // TODO: Make this dynamic
+    String path = "config/residential_ev.xml";
```

Comment:
I guess we can get this somehow from the classpath?

Reply:
Anything in the config dir is on the classpath.

Reply:
And if it's a .xml or a .properties file, config will see it.

Reply:
So the path can be a @ConfigurableValue